presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -203,6 +203,10 @@ Property Name                                Description
 ``hive.s3.signer-type``                      Specify a different signer type for S3-compatible storage.
                                              Example: ``S3SignerType`` for v2 signer type
 
+``hive.s3.path-style-access``                Use path style for S3 access. This is for S3-compatible storage
```

Comment:
Similar to the javadoc of `S3ClientOptions::setPathStyleAccess`, how about we say `Use path-style access for all requests to the S3-compatible storage.`?